functions/path-param-names.js

```diff
@@ -10,6 +13,8 @@ module.exports = (paths) => {
 
   // Dict to accumulate the parameter name associated with a path segment
   const paramNameForSegment = {};
+  // Dict to accumulate the parameter name associated with a path segment
```

Comment:
Comment should read "Dict to accumulate the *segment* name associated with a path segment"

Reply:
```suggestion
  // Dict to accumulate the segment name associated with a path parameter
```